mailing_list/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
+
+from . import views
+
+urlpatterns = [
+    url(r'^subscribe/$', views.subscribe, name='subscribe'),
+    url(r'^unsubscribe/$', views.unsubscribe, name='unsubscribe'),
```

Comment:
maybe make sure we don't conflict with other url names in openedx:
```suggestion
    url(r'^subscribe/$', views.subscribe, name='fun_mailing_list_subscribe'),
    url(r'^unsubscribe/$', views.unsubscribe, name='fun_mailing_list_unsubscribe'),
```